### DISCUSSION TITLE
Live Photos suddenly showing up as Videos instead

### DISCUSSION THREAD

**Discussion by aniqueta** [photoprism/photoprism#2411] [Q&A] [answered]
I'm not sure what I may have accidentally changed in my configuration, but Live Photos are now being imported as videos. Any idea what this could be? 

The logs show the Live Photos importing and stacking correctly. In the web interface, they are stacked correctly. The JPG from the HEIC of the Live Photo is the root of the stack. The JPG generated from the MOV and the HEIC itself could be selected as primary, but doing so has no change. I am able to change the type manually from Video to Live, and everything works normally. 

In terms of why they didn't import that way, the only thing that I think may be strange is that each index produces this database error: 
```
level=error msg="sql: Scan error on column index 32, name \"file_chroma\": converting driver.Value type []uint8 (\"133\") to a int8: value out of range"
```

I don't think this database issue could be related? Unless it's interrupting some other action? 

If it isn't that, then I think it's a configuration issue, even though I cannot recall making any change related to this, since old photos that previously imported correctly, when reimported, show up as videos as well. Moreover, I've tried older builds of PhotoPrism, even though this issue emerged without an upgrade to a new build. In case it's a permissions issue on my originals folder, I also granted full permissions without any effect. 

Any ideas? Happy to share logs and Live Photos examples, but since this also affects imported Live Photos previously imported correctly on the same build, I think it's a configuration issue on my end.

Thank you for the guidance!

**Comment by graciousgrey** [comment, marked as answer]
Do you have exiftool disabled?

**Comment by aniqueta** [comment]
Thanks for pointing me to this. In trace and debug, there was no message about exiftool. I'm running from source on macOS, and it looks like the auto-updated exiftool is broken in the Homebrew package manager and pending a fix.  Thanks again.
https://github.com/Homebrew/homebrew-core/pull/103182
